src/reducers/columnAnchor.js

```diff
@@ -0,0 +1,185 @@
+/**
```

Comment:
A copy-paste of [scrollAnchor.js](https://github.com/schrodinger/fixed-data-table-2/blob/3cc36b020c789a4f766fdc5c8042b71dc99ab410/src/reducers/scrollAnchor.js) but for columns.

*Recap:*
`scrollAnchor` calculates the "anchor"  for vertical scrolling. 
The anchor is simply an object that serves as the initial coordinates used for tracking scrolling. 
It includes the index of the first/last visible column in the viewport.
It also includes an offset field based on how far the row/column is cutoff from the viewport.